src/group.py

```diff
@@ -115,13 +123,13 @@ def start_move(self):
 
         self.path_ids = []
         # DRAW THE PATH SQUARES HERE
-        for x, y in self.cur_path:
-            id = self.canvas.create_rectangle(int(x) * utility.CELL_SIZE,
-                                              int(y) * utility.CELL_SIZE,
-                                              int(x) * utility.CELL_SIZE + utility.CELL_SIZE,
-                                              int(y) * utility.CELL_SIZE + utility.CELL_SIZE,
-                                              fill=self.color, width=0)
-            self.path_ids.append(id)
+        # for x, y in self.cur_path:
```

Comment:
Why did this get commented out?

Reply:
I think that the paths the group's took stayed on screen even after they stopped moving